apps/trustlab/src/payload/access/checkRole.js

```diff
@@ -0,0 +1,8 @@
+export const checkRole = (roles, user) => {
+  if (!user) {
+    return false;
+  }
+  return roles.includes(user.role);
```

Comment:
Or just:
```suggestion
  return roles.includes(user?.role);
```

`roles` doesn't include `undefined`.